src/layouts/base-layout/index.tsx

```diff
@@ -60,11 +60,42 @@ const BaseLayout = ({
 	useScrollPercentageAnalytics()
 	const [showSkipLink, setShowSkipLink] = useState(false)
 
+	// TODO: remove after HashiConf 2024
+	// start of code to remove
+	const [isDuringHashiConf, setIsDuringHashiConf] = useState(false)
+
+	useEffect(() => {
+		const now = new Date().getTime() // current date & time
```

Comment:
@heatlikeheatwave we have a timezone bug here. Namely we are getting a random timezone but then comparing it to PST, which could mean they are already in Hashiconf time when Hashiconf hasn't even started yet. We need to get the users timezone in PST to accurately compare to PST.

So now should be:
`const now = new Date().toLocaleString('en-US', { timeZone: 'America/Los_Angeles' })`

Reply:
Actually that is just going to just return the string. I think we should import something like: https://day.js.org/en/ and use it's date manipulation functions as that is going to make comparing across timezones easier.

Reply:
Shouldn't it be EDT since the conference is in Boston?

Reply:
Also, great catch with this!

Reply:
Dates. Why you so hard. 😢

Reply:
Y'all are making it happen with more 👁️ on the code.

Reply:
@RubenSandwich I tested this again, and I don't think this has the bug you described. I changed my location and time on my machine to west coast and europe; the visibility of the banner worked as expected. What am I missing that you're seeing? 

I do agree that the pacific time zone (.e.g -8:00 in the time stamp) can be changed. That was a mistake on my part.